### PR TITLE
Build out specs

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Build out specs with use with curation_concerns version 0.3.0.

### CODE REVIEW

File: app/models/concerns/image_behavior.rb [hunk 1]
```diff
@@ -5,9 +5,18 @@ module ImageBehavior
     type [Hydra::PCDM::Vocab::PCDMTerms.Object,
           Hydra::Works::Vocab::WorksTerms.GenericWork,
           "http://projecthydra.org/geoconcerns/models#Image"]
-    filters_association :members, as: :image_files, condition: :image_file?
-    filters_association :members, as: :metadata_files, condition: :external_metadata_file?
-    filters_association :members, as: :rasters, condition: :raster?
+  end
+
+  def image_file
```
Comment (drh-stanford) [inline app/models/concerns/image_behavior.rb]: why aren't use using `filters_association` anymore?

File: config/jetty.yml [hunk 2]
```diff
@@ -4,3 +4,4 @@ default:
   java_opts:
     - "-Xmx512m"
     - "-XX:MaxPermSize=128m"
+    - "-Djetty.host=localhost"
```
Comment (drh-stanford) [inline config/jetty.yml]: why is this needed? doesn't `jetty.host` default to localhost in hydra-jetty?

**Comment by drh-stanford** [comment]
the specs look great... i put a couple comments inline but otherwise i'm :+1:

**Comment by drh-stanford** [comment]
also, re: process, for PRs we can just use branches in this repo... no need to fork the repo into your own space... we should all have permissions now to push to the repo.

**Comment by jrgriffiniii** [comment]
@drh-stanford Regarding `filters_association`, this approach seems to have been abandoned in Hydra::Works:

https://github.com/projecthydra-labs/hydra-works/blob/master/lib/hydra/works/models/concerns/collection_behavior.rb#L22

I can work to resolve your initial comment, as this was likely something put into place for my Vagrant environment.

**Comment by eliotjordan** [comment]
:+1: Looks good. Glad to have some tests back.

**Comment by eliotjordan** [comment]
Also agree with @drh-stanford about branching from the repo.
